ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -77,6 +113,62 @@ objects:
         secretName: ${JOB_NAME_SAFE}-cluster-profile
 
     containers:
+    - name: egress-proxy
+      command:
+      - /bin/bash
+      - -c
+      args:
+      - |
+        #!/bin/bash
+        set -euo pipefail
+
+        if [[ "${ENABLE_PROXY}" != "true" ]]; then
+          exit 0
+        fi
+
+        trap 'rc=$?; if test "${rc}" -ne 0; then touch /tmp/shared/exit; fi; exit "${rc}"' EXIT
+        trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+        # this is just a stop gap -- we want to have a secret created to house this
+        echo -e 'admin:$apr1$JOHdRB4Z$S7zUn2dtdxlXLSgWdlFp40' > /tmp/passwords
+
+        mkdir /tmp/artifacts/squid
+
+        cat > /tmp/squid.conf <<EOF
+        http_port 3128
```

Comment:
Instead of translating from 443 at the route/ingress to 3128 here, should we just use `https_port 443` and then use 443 for the service and route too?